### PR TITLE
[WF-138] UV Migration Temporal Repos

### PR DESCRIPTION
**Pull request by arjun11-malik**
## Description

---
## Notes for Reviewers (optional)

---
## Checklist (all that apply)
- [ ] Unit tests added or updated
- [ ] Integration tests added or updated
- [ ] Documentation updated

### CODE REVIEW

File: .github/workflows/integration_test.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 jobs:
   integration-tests:
-    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    uses: arjun11-malik/operator-workflows/.github/workflows/integration_test.yaml@main
     secrets: inherit
     with:
-      channel: 1.28-strict/stable
+      provider: k8s
+      use-canonical-k8s: true
+      channel: edge
```
Comment (shayancanonical) [inline .github/workflows/integration_test.yaml]: suggestion, let's be explicit with the track (e.g. `latest/edge` or `1.28-strict/edge`

also, any reason why we're testing against k8s edge risk? and not the stable risk?

File: src/temporal_admin_k8s_operator.egg-info/PKG-INFO [hunk 2]
```diff
@@ -0,0 +1,41 @@
+Metadata-Version: 2.4
```
Comment (shayancanonical) [inline src/temporal_admin_k8s_operator.egg-info/PKG-INFO]: does the `.egg_info` directory need to be included in the repo?

File: .github/workflows/integration_test.yaml (hunk 1 above)
Comment (arjun11-malik) [reply .github/workflows/integration_test.yaml]: i have made this change in the [CI pr](https://github.com/canonical/temporal-k8s-operator/pull/102), i can rebase after that gets merged.

File: src/temporal_admin_k8s_operator.egg-info/PKG-INFO (hunk 2 above)
Comment (arjun11-malik) [reply src/temporal_admin_k8s_operator.egg-info/PKG-INFO]: no, I added to .gitignore and removed the directory from git. [3fed30d](https://github.com/canonical/temporal-admin-k8s-operator/pull/42/commits/3fed30d038c5aa60929d82b32cf772aea96c9065)

**Review by shayancanonical — APPROVED** [review]
great work!
